external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDelegationTokenSuite.scala

```diff
@@ -62,7 +62,7 @@ class KafkaDelegationTokenSuite extends StreamTest with SharedSparkSession with
     }
   }
 
-  test("Roundtrip") {
+  ignore("Roundtrip") {
```

Comment:
Oh sorry..we need to also disable `beforeAll/afterAll`. I'll make a follow-up right now.

Reply:
Just wanted to say :)

Reply:
Hi @gaborgsomogyi , after taking a look of `BeforeAndAfterAll`, I think it should be OK due to:

```
/**
   * Flag to indicate whether to invoke beforeAll and afterAll even when there are no tests expected.
   *
   * <p>
   * The default value is <code>false</code>, which means beforeAll and afterAll will not be invoked 
   * if there are no tests expected. Whether tests are expected is determined by invoking <code>expectedTestCount</code> passing in
   * the passed filter. Because this count does not include tests excluded based on tags, such as ignored tests, this prevents
   * any side effects in <code>beforeAll</code> or <code>afterAll</code> if no tests will ultimately be executed anyway.
   * If you always want to see the side effects even if no tests are expected, override this <code>val</code> and set it to true.
   * </p>
   */
  val invokeBeforeAllAndAfterAllEvenIfNoTestsAreExpected = false

```
And I tested locally to verify it. So, we're temporary safe by now. But please be aware of it as it can be easily break when there's another test. And, of course, the later fix could also get rid of it.

Reply:
Ah, so we're safe. Thanks for investigation; I was about to point this out too.